app/views/descriptions/show.html.haml

```diff
@@ -1,18 +1,14 @@
 #overlay_container
-  - if @description.title
-    %h1= raw t(@description.title)
+  %h1= I18n.t(@chart.title_for_description)
 
   %p
-    - if @description.content
-      = raw @description.content
+    - if @description.blank?
+      No description available yet
```

Comment:
The descriptions for charts are optional. You are right about the fact that this shouldn't exist. In fact I can just remove the whole if statement, because the check for redendering the info button on a chart has become more strict.